src/uwtools/tests/config/test_tools.py

```diff
@@ -607,9 +615,8 @@ def test__print_config_section_yaml_not_dict():
 def test__realize_config_input_setup_ini_cfgobj():
     data = {"section": {"foo": "bar"}}
     cfgobj = INIConfig(config=data)
-    input_obj, input_format = tools._realize_config_input_setup(input_config=cfgobj)
+    input_obj = tools._realize_config_input_setup(input_config=cfgobj)
```

Comment:
`_realize_config_input_setup()` no longer returns an input format value, so this is removed here and in multiple similar cases below.